manifests/03_credentials_request_gcp.yaml

```diff
@@ -17,12 +17,22 @@ spec:
   providerSpec:
     apiVersion: cloudcredential.openshift.io/v1
     kind: GCPProviderSpec
-    predefinedRoles:
-      # FIXME: find a replacement for instanceAdmin, since the CSI driver
-      # only needs "compute.instances.[get|attachDisk|DetachDisk]"
-      - roles/compute.instanceAdmin
-      - roles/compute.storageAdmin
-      - roles/iam.serviceAccountUser
+    permissions:
+      - "storage.buckets.create"
+      - "storage.buckets.delete"
+      - "storage.buckets.get"
+      - "storage.buckets.list"
+      - "storage.objects.create"
+      - "storage.objects.delete"
+      - "storage.objects.get"
+      - "storage.objects.list"
+      - "storage.buckets.getIamPolicy"
```

Comment:
Why the **disk** CSI driver needs permission for buckets and objects?

And why there are no permissions for disks?

Reply:
I wanted to test what I found in our documentation, after we discussed this further this PR should rather change permissions so it matches to what upstream is using. I've updated the description with all relevant references. 

What is unclear to me still is the `editor` role - chances are it's not needed but we can evaluate it later when fine grained permissions are defined upstream.